tests/microbenchmarks.py

```diff
@@ -192,6 +200,7 @@ def test_dict_get(test_case):
             dict.get("a")
     """), timed=True)
 
+
 def test_dict_set(test_case):
     print("Running", "test_dictionary_set")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils